x-pack/plugins/monitoring/common/types/es.ts

```diff
@@ -142,6 +142,14 @@ export interface ElasticsearchIndexStats {
   };
 }
 
+export interface ElasticsearchLogstashStatePipeline {
```

Comment:
I've created this type to improve the return type readability on `getPipeline` function